app/graphql/types/title_language_preference.rb

```diff
@@ -0,0 +1,5 @@
+class Types::TitleLanguagePreference < Types::BaseEnum
+  value 'CANONICAL', value: 'canonical'
+  value 'ROMANIZED', value: 'romanized'
+  value 'LOCALIZED', value: 'localized'
```

Comment:
These should get descriptions!
```suggestion
  value 'CANONICAL', 'Prefer the most commonly-used title for media', value: 'canonical'
  value 'ROMANIZED', 'Prefer the romanized title for media', value: 'romanized'
  value 'LOCALIZED', 'Prefer the localized title for media', value: 'localized'
```